adapter/internal/oasparser/envoyconf/constants.go

```diff
@@ -23,6 +23,11 @@ const (
 	tracingClusterName      string = "wso2_cc_trace"
 )
 
+// Clusters predefined on envoy
+const (
+	tokenCluster = "token_cluster"
```

Comment:
would be ideal if we can rename it to `ext_authz_http_cluster` like term since we are not just using it for testkey endpoint rather for jwks endpoint too.